src/daft-sql/src/functions.rs

```diff
@@ -242,14 +243,21 @@ impl<'a> SQLPlanner<'a> {
         // assert using only supported features
         check_features(func)?;
 
+        fn get(
+            fns: &Lazy<SQLFunctions>,
+            name: impl AsRef<str>,
+        ) -> SQLPlannerResult<Arc<dyn SQLFunction>> {
+            let name = name.as_ref();
+            let fn_match = fns.get(name).ok_or_else(|| {
+                PlannerError::unsupported_sql(format!("Function `{}` not found", name))
+            })?;
+            Ok(fn_match.clone())
```

Comment:
Minor nit: could you get away with this instead?
```suggestion
            fns.get(name).ok_or_else(|| {
                PlannerError::unsupported_sql(format!("Function `{}` not found", name))
            })
```

Reply:
Yup! That works too. Will update in the coming commit.

Reply:
Updated.